source/_integrations/vicare.markdown

```diff
@@ -24,51 +25,86 @@ ha_codeowners:
 The **Viessmann ViCare** {% term integration %} lets you control [Viessmann](https://www.viessmann.com) devices via the Viessmann ViCare (REST) API.
 Most recent network-connected Viessmann heating devices (e.g., gas boilers) should be supported.
 
-There is currently support for the following device types within Home Assistant:
+## Prerequisites
 
-- [Climate](#climate) (Heating)
-- [Water heater](#water-heater) (Domestic hot water)
-- [Sensor](#sensor) (Sensor)
-- [Button](#button) (Button)
-- [Number](#number)
+You will need to sign in on the [Viessmann developer portal](https://app.developer.viessmann.com/) with **your existing ViCare app user credentials**.
 
-{% include integrations/config_flow.md %}
-
-Use your ViCare app login credentials for *username* and *password*.
-
-The required *client ID* can be obtained from the [Viessmann Developer Portal](https://app.developer.viessmann.com/):
-1. Log in with **your existing ViCare app username and password**.
-2. On the developer dashboard, select **Add** in the **Clients** section.
-3. Create a new client using the following data:
+Create a new API client by selecting **Add** in the **Clients** section on the developer dashboard with the following settings:
    - Name: `HomeAssistant`
-   - Google reCAPTCHA: Disabled
+   - Google reCAPTCHA: `disabled`
    - Redirect URIs: `vicare://oauth-callback/everest`
-4. Find the **Client ID** in the **Clients** section on the developer dashboard.
 
-The `heating_type` can either be `auto` to automatically find the most suitable type for your device or one of `gas`, `oil`, `pellets`, `heatpump`, `fuelcell`, `hybrid`.
+Copy the **Client ID** in the **Clients** section on the developer dashboard for the setup in Home Assistant.
 
-Important: the redirect URI that you configure requires that you perform the initial setup on a device that has the ViCare application installed. If your device does not know how to handle the `vicare://` URL, you will receive an **Invalid credentials** notification and the setup procedure will fail. This means: install the ViCare app on your phone and set up the {% term integration %} from your phone.
+{% important %}
+Setup the {% term integration %} from your device (phone) where you have the ViCare app installed. Otherwise your device does not know how to handle the `vicare://` redirect URL and you will receive an **Invalid credentials** notification and the setup procedure will fail.
```

Comment:
```suggestion
You have to setup the {% term integration %} from your device (phone) where you have the ViCare app installed. Otherwise, your device does not know how to handle the `vicare://` redirect URL, and you will receive an **Invalid credentials** notification and the setup procedure will fail.
```